metrics/web/test/features/auth/domain/entities/authentication_exception_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:metrics/features/auth/domain/entities/auth_error_code.dart';
+import 'package:metrics/features/auth/domain/entities/authentication_exception.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("AuthenticationException", () {
+    test(
+      "constructor creates the instance with unknown error code when the null is passed",
+      () {
+        const authException = AuthenticationException(code: null);
```

Comment:
```suggestion
        final authException = AuthenticationException(code: null);
```
see https://github.com/dart-lang/sdk/issues/38934
and we can continue our discussion in Teams